seed/migrations/0108_auto_20190730_1212.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.21 on 2019-07-30 19:12
+from __future__ import unicode_literals
+
+from django.db import connection, migrations, models
+import django.utils.timezone
+
+
+def forwards(apps, schema_editor):
+    property_sql = (
+        "UPDATE seed_propertystate " +
+        "SET created = seed_propertyauditlog.created, updated = seed_propertyauditlog.created " +
```

Comment:
Note: In order to populate these fields with a reasonable base value, I make an attempt to update these new `created` and `updated` -State fields with the value from the corresponding -AuditLog. 

If a -State doesn't have an -AuditLog, the "current' time of when the migration is run is used for both values. I'm only aware of this (a -State without an -AuditLog) happening is when the preliminary, extra_data-only records are created during import.